CHANGELOG.md

```diff
@@ -1,4 +1,96 @@
-## 2.18.2 (Unreleased)
+## 2.19.0 (Unreleased)
+### Notes:
+- Supported Controller version: **UserConnect-6.4.2416**
```

Comment:
version will likely change before we release. At the moment of writing, it is 6.4.2433